tests/tracer/test_trace_utils.py

```diff
@@ -197,11 +197,25 @@ def test_activate_distributed_headers_no_headers(int_config):
 
 
 def test_sanitized_url_in_http_meta(span, int_config):
+    FULL_URL = "http://example.com/search?q=test+query#frag?ment"
```

Comment:
We might want to set this us as `pytest.mark.parametrize` so we can keep building up examples

Another example I'd like to see is a "path only" url: `/some/path/here?qs=value`

maybe some examples without any query string at all, some with only fragment, some with one query string, some with multiple query string values, ones with too many `?` or `#`, etc

Reply:
I think here we only need to test that `set_http_meta` works well with `int_config.trace_query_string`. The result of `strip_query_string` could be checked with dedicated test cases.